src/components/molecules/Common/AssetModal/AssetContainer/index.tsx

```diff
@@ -166,35 +209,34 @@ const AssetContainer: React.FC<Props> = ({
 };
 
 const Wrapper = styled.div`
-  height: 558px;
   width: 100%;
 `;
 
-const AssetWrapper = styled(Flex)`
-  height: 425px;
+const AssetWrapper = styled.div<{ isHeightFixed?: boolean }>`
```

Comment:
Why did you turn this into a div and then place flex on it? It was originally using our Flex component. Any limitations you found on the Flex component that made you switch to a div?

Reply:
yes, I feel when I develop, if I use a flex component which is a little hard to use, the properties is different from native css, and because of using flex-component , I made some mistakes. Here of course I can still use flex-component, so you want to use back the flex-component ?

Reply:
If you find it hard to use, the team can talk more about its usefulness. For now though we have been trying to go in the direction of using it when appropriate. If it is really hard to use perhaps discuss with @HideBa when we have time.

Reply:
and also the flex-component don't have align-content property . align-items and align-content are different ... I guess

Reply:
but I remembr @HideBa is agree with us can not use flex-component , right? Or we must ? @HideBa @KaWaite ?

Reply:
and here, because I need to get the width of div, it's just difficult using Flex-component here